conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -117,10 +117,11 @@ else
         --extra-meta flow_run_id="$flow_run_id" remote_url="$remote_url" sha="$sha"
 
     {%- else %}
-
+    export SYSTEM_VERSION_COMPAT=0
```

Comment:
Please build rattler with a `11.0` SDK and remove this here.

Reply:
How would we do that on conda-forge? Aren't we specifically using an older one?

Reply:
https://github.com/conda-forge/rattler-build-feedstock/pull/87/files